apps/web/.env

```diff
@@ -1,2 +1,8 @@
 NEXT_PUBLIC_MM_API_URL=https://linked-pool.pancakeswap.com/quote-service
-NEXT_PUBLIC_QUOTING_API=https://swap-quoting.pancakeswap.com/quoting-service
\ No newline at end of file
+NEXT_PUBLIC_QUOTING_API=https://swap-quoting.pancakeswap.com/quoting-service
+MOONPAY_URL=https://buy-sandbox.moonpay.com?apiKey=pk_test_1Ibe44lMglFVL8COOYO7SEKnIBrzrp54
+MOONPAY_TEST_SECRET_KEY=sk_test_7zfPNfcZdStyiktn3lOJxOltGttayhC
+MOONPAY_TEST_API_KEY=sk_test_7zfPNfcZdStyiktn3lOJxOltGttayhC
```

Comment:
These keys are for testing purpose right?

Reply:
yes. also still have this in draft. need work on for few more days

Reply:
done